internal/common/voting/election.go

```diff
@@ -86,8 +86,114 @@ func (e *Election) CloseBallot() shared.ClientID {
 }
 
 func (e *Election) bordaCountResult() shared.ClientID {
-	// TODO implement Borda count winner selection method.
-	return e.pluralityResult()
+	// Implement Borda count winner selection method
+	//(may need to modify if methods design is changed)
+	var candidatesNumber int = 0
+	var islandsNumber int
+	var votesLayoutElect map[int][]int
```

Comment:
if this represents the islands votes then make it a map[shared.ClientID][]int for standarization

Reply:
it wouldn't be used as output of this function, just convert the votesSliceSquare to map[int][]int for calculation